standard/PortalStatistics.lua

```diff
@@ -0,0 +1,1131 @@
+	table.sort(opponentData, function(a, b) return earningsFunction(a) > earningsFunction(b) end)
+
+	local opponentPlacements = StatisticsPortal._cacheOpponentPlacementData(args)
+
+	local tbl = mw.html.create('table')
+		:addClass('wikitable wikitable-striped wikitable-bordered sortable')
+		:css('margin-left', '0px')
+		:css('margin-right', 'auto')
+		:css('width', '100%')
+
+	tbl:node(StatisticsPortal._earningsTableHeader(args))
+
+	for opponentIndex, opponent in ipairs(opponentData) do
+		local opponentDisplay
+		local earnings = earningsFunction(opponent)
+
+		if opponentIndex > args.limit or earnings < 1000 then break end
+
+		if args.opponentType == Opponent.team then
+			opponentDisplay = OpponentDisplay.BlockOpponent{
+				opponent = {template = opponent.template, type = Opponent.team},
+				teamStyle = 'standard',
+			}
+		else
+			opponentDisplay = OpponentDisplay.BlockOpponent{
+				opponent = StatisticsPortal._toOpponent(opponent),
+			}
+		end
+		local placements = opponentPlacements[opponent.pagename] or {}
+		tbl:node(StatisticsPortal._earningsTableRow(args, placements, earnings, opponentIndex, opponentDisplay))
+	end
+
+	return mw.html.create('div'):addClass('table-responsive'):node(tbl)
+end
+
+
+--[[
+Section: Player Age Table Breakdown
+]]--
+
+
+function StatisticsPortal.playerAgeTable(args)
+	args = args or {}
+	args.earnings = args.earnings and tonumber(args.earnings) or 500
+	args.limit = args.limit and tonumber(args.limit) or 20
+	args.order = args.order and 'birthdate ' .. args.order or 'birthdate desc'
+
+	local conditions = ConditionTree(BooleanOperator.all)
+		:add{ConditionNode(ColumnName('birthdate'), Comparator.neq, '')}
+		:add{ConditionNode(ColumnName('birthdate'), Comparator.neq, EPOCH_DATE)}
+		:add{ConditionNode(ColumnName('deathdate'), Comparator.eq, EPOCH_DATE)}
+		:add{ConditionNode(ColumnName('earnings'), Comparator.gt, args.earnings)}
+
+	if Logic.readBool(args.isActive) then
+		conditions:add{ConditionNode(ColumnName('status'), Comparator.eq, 'Active')}
+	end
+
+	if Logic.readBool(args.playersOnly) then
+		local typeConditions = ConditionTree(BooleanOperator.any)
+		typeConditions:add{
+			ConditionNode(ColumnName('type'), Comparator.eq, 'player'),
+			ConditionNode(ColumnName('type'), Comparator.eq, 'Player'),
+		}
+		conditions:add{typeConditions}
+	end
+
+	local playerData = StatisticsPortal._getPlayers(args.limit, conditions:toString(), args.order)
+
+	local tbl = mw.html.create('table')
+		:addClass('wikitable wikitable-striped sortable')
+		:css('margin-left','0px')
+		:css('margin-right','auto')
+
+	tbl:tag('tr')
+		:tag('th'):wikitext('ID'):addClass('unsortable'):done()
+		:tag('th'):wikitext('Age')
+
+	for _, player in ipairs(playerData) do
+		local birthdate =  DateExt.readTimestamp(player.birthdate)
+		local yearAge = math.floor(os.difftime(TIMESTAMP, birthdate)/(DAY_SECONDS * 365.25))
+		local dayAge = Math._round(Math._mod(math.floor(os.difftime(TIMESTAMP, birthdate)/DAY_SECONDS), 365.25), 0)
```

Comment:
```suggestion
		local age = os.date('*t', os.difftime(TIMESTAMP, birthdate))
		local yearAge = age.year
		local dayAge = age.yday
```
Think this should be correct written and actually produce the intended result (instead of being slightly off occupationally on both age and day)